pkg/cloudprovider/aws/aws.go

```diff
@@ -50,6 +50,20 @@ func verifyIfErrorOccured(apiErr error, expectedMessage string) (bool, error) {
 	return false, apiErr
 }
 
+func verifyIfErrorOccuredWithDefaults(apiErr error, expectedMessage string) (bool, error) {
+	skip_errs := []string{
+		"is not in correct state",
+	}
+	if awsErr, ok := apiErr.(awserr.Error); ok {
```

Comment:
Can you bake this functionality into the `verifyIfErrorOccured` function? Atm you're duplicating functionality. The equivalent can be achieved using a variadic argument, or just by providing a list of expected error messages. In this function you can simply declare them all and pass them to it, this function just happens to provide the specified defaults.

Reply:
Also it's worth taking the opportunity to fix the typo in the function name, `verifyIfErrorOccured` -> `verifyIfErrorOccurred `